packages/react/src/hooks/utils/query/useBaseQuery.ts

```diff
@@ -28,7 +30,7 @@ export function useBaseQuery<
   >,
   Observer: typeof QueryObserver,
 ) {
-  const queryClient = useQueryClient({ context: options.context })
+  const queryClient = useQueryClient({ context: queryClientContext })
```

Comment:
Can we instead leave this as is (would rather not touch this file as it's forked from RQ), and instead place this inside `utils/query/useQuery` and `utils/query/useInfiniteQuery`?